src/main/scala/org/globalforestwatch/layers/GrossCumulAbovegroundRemovalsCo2.scala

```diff
@@ -0,0 +1,16 @@
+package org.globalforestwatch.layers
+
+import org.globalforestwatch.grids.GridTile
+
+case class GrossCumulAbovegroundRemovalsCo2(gridTile: GridTile, model: String="standard")
+  extends FloatLayer
+    with OptionalFLayer {
+  //      val model_suffix = if (model == "standard") "" else s"__$model"
+  val model_suffix: String = if (model == "standard") "standard" else s"$model"
```

Comment:
Codacy found an issue: [Field name 'model_suffix' does not match the regular expression '^[a-z][A-Za-z0-9]*$'](https://app.codacy.com/manual/thomas-maschler/gfw_forest_loss_geotrellis/pullRequest?prid=6075981)